lite/requirements.txt

```diff
@@ -7,3 +7,4 @@ jupyterlite
 nbformat
 pkginfo
 pyviz_comms
+jupyterlite-open-url-parameter
```

Comment:
Looks like this should be `jupyterlab-open-url-parameter` here.

Reply:
Indeed, good catch thanks.

Reply:
I assume you followed this: https://github.com/jupyterlite/jupyterlite/blob/76cd0496add7494bec9a749b2044e28074f60781/docs/howto/content/open-url-parameter.md.

@jtpio Does that mean that the document should be updated?

Reply:
Probably this link points to an old version of the docs?

The docs look good here: https://jupyterlite.readthedocs.io/en/latest/howto/content/open-url-parameter.html